core/src/main/java/io/questdb/griffin/SqlCompilerImpl.java

```diff
@@ -1432,6 +1438,11 @@ private void compileAlter(SqlExecutionContext executionContext, @Transient CharS
                     } else {
                         throw SqlException.$(lexer.lastTokenPosition(), "'=' expected");
                     }
+                } else if (SqlKeywords.isTtlKeyword(tok)) {
+                    // TODO: validate that the table is partitioned
+                    int ttlHoursOrMonths = SqlParser.parseTtlHoursOrMonths(lexer);
+                    compiledQuery.ofAlter(alterOperationBuilder.ofSetTtlHoursOrMonths(
```

Comment:
nit: function call formatting - very casual, should be either all args on one line or one arg per line, otherwise readability suffers